### ISSUE TITLE
OCIL clause of package installed is inverted

### ISSUE THREAD

**Issue by ggbecker** [ComplianceAsCode/content#8513]
I believe OCIL questions should always be asked in a negative form, for example

`Is the case that `**the package is not installed**`?`

https://github.com/ComplianceAsCode/content/blob/3753283b2f00f3dcbf396bd5f66e7053e390dc12/shared/macros.jinja#L694

https://github.com/ComplianceAsCode/content/blob/3753283b2f00f3dcbf396bd5f66e7053e390dc12/shared/macros.jinja#L708

This leads to a wrong question to be rendered in the STIG Spreadsheet creation for example

```
Run the following command to determine if the  chrony  package is installed:
 $ rpm -q chrony 

If the package is installed, then this is a finding.
```

**Update1**: This macro is mostly used for package removed rules. But some rules that require installing the package are also using it and leads to wrong ocil data.

**Comment by Mab879** [comment]
It should be a negative.  On a compliant system the OCIL question "Is it the case that" should be false statement.

**Comment by ggbecker** [comment]
Fixed via #8523
